### DISCUSSION TITLE
2.3.0 version webhook repeat send

### DISCUSSION THREAD

**Discussion by YuZongYangHi** [jertel/elastalert2#1362] [Q&A] [answered]
hello, long time no see. Recently we encountered a problem. The elastalert2 service of version 2.3.0 was deployed in a container. It has been running for a long time. Recently, we found that webhooks are sent repeatedly. Does this problem exist in this version?

**Comment by YuZongYangHi** [comment]
@jertel

**Comment by jertel** [comment, marked as answer]
Hello. From your post there's not enough information to determine if it's duplicate alerts from a single rule run, or if each rule run is triggering an alert. Please review #11 and provide the requested information. We especially need debug logs showing this behavior. 

Also, I'm curious why you opened this as an issue instead of a discussion. Is the "STOP" message on the New Issue template screen not easily understood, or perhaps there's a language barrier/translation issue? 

![image](https://github.com/jertel/elastalert2/assets/16639611/aa62c57f-e528-4690-ad53-22e1e779ac2e)

You're feedback could help us improve the process, so thanks for assisting!
